### PR TITLE
Update page-conf.php

### PR DESCRIPTION
**Pull request by jospoortvliet**
Add streaming link to our conf page

### CODE REVIEW

File: page-conf.php [hunk 1]
```diff
@@ -62,7 +63,7 @@
 			</ul></p>
 			<p class="section--paragraph revealOnScroll">On Friday the 25th we have a <a class="hyperlink" href="https://nextcloud.com/blog/focus-friday-during-the-nextcloud-conference/">special program with a focus on enterprise and large scale Nextcloud usage.</a></p>
 			<p class="section--paragraph revealOnScroll">Check out <a class="hyperlink" href="https://eventyay.com/e/de88e486/schedule/">the full program</a>!</p>
-<!-- 			<p class="section--paragraph">Stay tuned for the program for this year!</p> -->
+        <p class="section--paragraph"><a class="hyperlink" href="`https://player.cloud.wowza.com/hosted/rpfqrcyp/player.html">Follow the talks streamed online</a></p>
```
Comment (LukasReschke) [inline page-conf.php]: href="`htt this URL won't work

File: page-conf.php [hunk 2]
```diff
@@ -27,6 +27,7 @@
 			<p class="section--paragraph revealOnScroll"><a href="https://eventyay.com/e/de88e486/" class="btn btn-lg btn-primary">Register for free</a> <a href="https://eventyay.com/e/de88e486/cfs/" class="btn btn-lg btn-primary">Submit a talk <i class="icon-arrow-circle-o-right icon"></i></a></p>
 			<p class="section--paragraph revealOnScroll"><a href="https://eventyay.com/e/de88e486/schedule/" class="hyperlink" >See the program</a></p>
 			<p class="section--paragraph">Find <a class="hyperlink" href="/confinfo">practical information for visitors here</a>.</p>
+            <p class="section--paragraph"><a class="hyperlink" href="`https://player.cloud.wowza.com/hosted/rpfqrcyp/player.html">Follow the talks streamed online</a></p>
```
Comment (LukasReschke) [inline page-conf.php]: href="`htt, this URL won't work

**Comment by jospoortvliet** [comment]
@MariusBluem what spacing?
@LukasReschke fixed.

**Comment by MariusBluem** [comment]
62ba3ec1-663b-45cd-852f-aa5219036d25

**Comment by jospoortvliet** [comment]
That is already fixed.

**Comment by jospoortvliet** [comment]
Tnx for reviewing!

**Pull request merged**